### ISSUE TITLE
authToken: undefined in context

### ISSUE THREAD

**Issue by stolinski** [flyblackbird/apollo-accounts#1]
Hey @lorensr 
I'm started implementing this and I'm having a hard time getting context set properly. `Accounts.getUser()` is returning the correct user info client side, so I know the login worked.

Setting context with:

```
const apolloServer = new ApolloServer({
  schema,
  context: ({ req }) => accountsContext(req),
})
```

My context is looking like: 
```
{ authToken: undefined,
  userAgent:
   'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3809.132 Safari/537.36',
  ip: '::1',
  _extensionStack:
   GraphQLExtensionStack {
     extensions: [ [EngineReportingExtension], [CacheControlExtension] ] }
```

Any thoughts?

**Comment by lorensr** [comment]
Hey Scott, thanks for reporting!

Reproduced with a valid `accounts-access-token` header! Must be a bug in `"@accounts/graphql-api": "0.3.0-beta.26"`. Current version of the module is `0.19.0`. We'll be upgrading to `0.19.0` at some point in the near future. In the meantime, if anyone wants to work on a PR, it would involve either forking `@accounts/graphql-api @ 0.3.0-beta.26` or upgrading to `0.19.0` on all `@accounts/*` packages (in both `server/` and `client/`)and updating the API accordingly.

**Comment by stolinski** [comment]
I'll have to investigate, but as an FYI, I'm having this exact same bug using a fully accounts-js setup without using apollo-account. https://github.com/accounts-js/accounts/issues/803

Not sure what the deal here is, because context within accounts-js is defined, but in my own resolvers is not.

**Comment by lorensr** [comment]
This was fixed by https://github.com/flyblackbird/apollo-accounts/commit/e30be119c04845d7cc0d5f1fd7b4a65da7f408a3 and published in `0.3.0-beta.26-1`
